aquadoggo/src/db/stores/blob.rs

```diff
@@ -10,14 +11,11 @@ use p2panda_rs::storage_provider::traits::DocumentStore;
 use sqlx::{query_scalar, AnyPool};
 
 use crate::db::errors::{BlobStoreError, SqlStoreError};
-use crate::db::query::{Field, Filter, Order, Pagination, Select};
+use crate::db::query::{Filter, Order, Pagination, PaginationField, Select};
 use crate::db::stores::query::{Query, RelationList};
 use crate::db::SqlStore;
 
-/// The maximum allowed number of blob pieces per blob.
-/// @TODO: do we want this? If so, what value should it be and we should add this to
-/// p2panda-rs blob validation too.
-const MAX_BLOB_PIECES: u64 = 10000;
+const BLOB_QUERY_PAGE_SIZE: u64 = 10;
```

Comment:
Think it's cool to paginate over blob pieces, it's an optimization but really nice for very large files 😎